webcompat/views.py

```diff
@@ -230,16 +230,17 @@ def create_issue():
         log.info('{ip} {url}'.format(
             ip=request.remote_addr,
             url=form['url'].encode('utf-8')))
-        # Checking blacklisted domains
-        if is_blacklisted_domain(urlparse.urlsplit(form['url']).hostname):
-            msg = app.config['IS_BLACKLISTED_DOMAIN'].format(form['url'])
-            flash(msg, 'notimeout')
-            return redirect(url_for('index'))
         # Check if the form is valid
         if not is_valid_issue_form(form):
             abort(400)
-        # Anonymous reporting
+        # Checking blacklisted domains
```

Comment:
Almost there. The comment should be after the `if form.get('submit_type') == PROXY_REPORT` as this `if` is about proxy reporting not blacklisting alone. :)

Also probably better to have the commit messages like:

```
Issue #2688 - Adds blacklist domain check only to anonymous reporter
```

* shorter.
* Says what it does.

Reply:
I have made the changes. Please review now :)